docs/design/historical_usage_tracking.md

```diff
@@ -45,23 +45,23 @@ Work to be tracked under [YUNIKORN-1628](https://issues.apache.org/jira/browse/Y
 
 ## Goals
 
--   Implementation of an event stream for an application, including:
-  -   State changes
-  -   Asks and allocation changes
+-   Implementation of an event stream for an application, including:  
+    -   State changes
+    -   Asks and allocation changes
 -   Implementation of an event stream for a node, including:
-  -   State changes
-  -   Allocation changes
+    -   State changes
+    -   Allocation changes
 -   Implementation of an event stream for a queue, including:
-  -   State changes
-  -   Usage changes
+    -   State changes
+    -   Usage changes
 -   Define a REST interface for event retrieval
 
 ## Non Goals
 
 -   Add a data store for the historical data
 -   Display the event information
 -   Rebuild data on recovery
-  -   Historical data will not be rebuild
+    -   Historical data will not be rebuild
```

Comment:
It seems this is unnecessary, right?

Reply:
Looks like it was intended to be at level-2, since it's already indented once before my change.

Reply:
Could you attach the screenshot for this change also?

Reply:
Okay.